styles.css

```diff
@@ -40,10 +40,17 @@ h2 label {
   padding: 20px 45px;
 }
 
-select {
-  display: block;
-  margin: 5px 0;
-  width: 150px;
+input, select {
```

Comment:
I'm not sure that all the inputs ever will have this styling... 🤔 
also my code style ocd says that if you have multiple rule declarations they each need to be on a separate line like
input,
select {
...

I would also like to keep attribution of where this styling came from if we can.